apps/emqx_resource/src/emqx_resource_validator.erl

```diff
@@ -39,7 +39,7 @@ enum(Items) ->
     end.
 
 required(ErrMsg) ->
-    fun(undefined) -> {error, ErrMsg};
+    fun(<<>>) -> {error, ErrMsg};
```

Comment:
There should be some configuration items whose value is empty binary.

Reply:
Empty binary configuration items can be without validator